WTWSMS/events/VIET_Jewish_Flavor_Events.txt

```diff
@@ -3,7 +3,7 @@
 
 #FLAVOR EVENTS
 
-#A bigoted courtier questions why we must have Jews in the court
+#A courtier questions why we must have Jews in the court
```

Comment:
It doesn't make it less of crap. It's a comment, not actual code. You've made several changes to integrated Mods before.